packages/console/src/pages/TenantSettings/Subscription/PlanComparisonTable/PlanQuotaKeyLabel/index.tsx

```diff
@@ -37,15 +42,23 @@ const planQuotaKeyPhraseMap: {
   auditLogsRetentionDays: 'audit_logs.retention',
   communitySupportEnabled: 'support.community',
   ticketSupportResponseTime: 'support.customer_ticket',
-  organizationsEnabled: 'organizations.organizations',
+  /**
+   * Todo @xiaoyijun [Pricing] Remove feature flag
+   */
+  organizationsEnabled: isDevFeaturesEnabled
+    ? 'organizations.monthly_active_organization'
+    : 'organizations.organizations',
 };
 
 type Props = {
   quotaKey: keyof SubscriptionPlanTable;
 };
 
 function PlanQuotaKeyLabel({ quotaKey }: Props) {
-  return <DynamicT forKey={`subscription.quota_table.${planQuotaKeyPhraseMap[quotaKey]}`} />;
+  const phraseKey = planQuotaKeyPhraseMap[quotaKey];
+  return (
+    cond(phraseKey && <DynamicT forKey={`subscription.quota_table.${phraseKey}`} />) ?? <div>-</div>
```

Comment:
The `div` might not be necessary?

Reply:
It's needed, but updated with react fragment.